### PR TITLE
Add back logging statement used by foxsec-pipeline, document others

### LINKED ISSUE #16369: Add logging statement at login time back
In e10f985cb2344e0e61140ee32a704c1e0304d7ac we removed the `Got profile {...}` log statement, but it turns out it's actually useful for the foxsec-pipeline, so let's add it back.

### PR DESCRIPTION
**Pull request by diox**
Fixes #16369

### CODE REVIEW

File: src/olympia/accounts/views.py [hunk 1]
```diff
@@ -303,6 +303,8 @@ def inner(self, request):
                     request, ERROR_NO_PROFILE, next_path=next_path, format=format
                 )
             else:
+                # The following log statement is used by foxsec-pipeline.
+                log.info('Logging in FxA user %s', identity['email'])
```
Comment (diox) [inline src/olympia/accounts/views.py:307]: Note: this is a reworded version of what was there before, with only the relevant info needed by foxsec-pipeline.

**Review by eviljeff — APPROVED** [review]
Relying on precise strings and comments just still seems fragile to me, but I don't have a better idea so 👍

File: src/olympia/accounts/views.py (hunk 1 above)
Comment (diox) [reply src/olympia/accounts/views.py:307]: https://github.com/mozilla-services/foxsec-pipeline/issues/510 goes with that change

Comment (willdurand) [reply src/olympia/accounts/views.py:307]: oops sorry about that, good call on adding comments for the future!

**Pull request merged**